python/chronos/src/bigdl/chronos/metric/forecast_metrics.py

```diff
@@ -129,3 +130,40 @@ def evaluate(metrics, y_true, y_pred, aggregate='mean'):
                     res = metric_func(y_pred, y_true)
                     res_list.append(res.numpy())
         return res_list
+
+    def get_latency(num_running, func, *args):
```

Comment:
I think we may have a default value for `num_running` here, such as 100, and a `**kargs` for func.
Here is an example code:
```python
>>> def func(arg1, *arg, times=None, **kwargs):
        print(arg1, arg, times, kwargs)
>>> func(50, 10, 20, times=100, other_param=90)
>>>  50 (10, 20) 100 {'other_param': 90}
```